src/vs/workbench/browser/parts/editor/editor.ts

```diff
@@ -41,6 +42,24 @@ export const DEFAULT_EDITOR_PART_OPTIONS: IEditorPartOptions = {
 	splitSizing: 'distribute'
 };
 
+export function computeEditorAriaLabel(input: IEditorInput, group: IEditorGroup | undefined, groupCount: number): string {
```

Comment:
Slight optimization: if you also allow to pass in the index of the editor (as `number | undefined`), the call to `isSticky` can use that info and does not have to loop over editors.

Reply:
Makes the whole thing a bit more ugly. Not a fan, especially since the textEditor.ts does not have an index.